### ISSUE TITLE
[Feature Request] show appropriate message in UI when query-service is not ready

### ISSUE THREAD

**Issue by prashant-shahi** [SigNoz/signoz#725]
## Is your feature request related to a problem? Yes.

It takes query-service sometime to be READY in kubernetes.
Currently, we see empty page with "Not Found" message.

![frontend-ui](https://user-images.githubusercontent.com/11138844/153814265-f9e6ee83-82f2-4251-bf3f-bc66e440d757.png)

## Describe the solution you'd like

It would be nice to have a UI with appropriate message to wait for query service to be ready and try again. Also, if it is not ready for sometime, check the health status and logs.

/cc @palash-signoz

**Comment by palash-signoz** [comment]
@prashant-shahi in this case api return 404 which is a valid case and render error component with error message

**Comment by prashant-shahi** [comment]
@palash-signoz Yes, you are right.
However, the message is unclear about what is "not found".

**Comment by pranay01** [comment]
@prashant-shahi We already have a not found page - would a message like this be appropriate? Do we expect this to get ready in 2-3 minutes or does it take more time?

`SigNoz is getting setup. Please check back in 2-3 mins... 
`
![query-service-not-ready](https://user-images.githubusercontent.com/504541/157002281-edd81ccc-bc04-47b7-a1a5-30c77f034c09.png)

**Comment by prashant-shahi** [comment]
yes, that UI looks good. Except for the sidebar on the left. They wouldn't work anyways if query-service is not ready yet.

Actually a status page UI would be nice: showing `READY` status of all SigNoz components.
- clickhouse
- query service
- otel collector
- otel collector metrics
